src/main/scala/io/iohk/ethereum/vm/GasFee.scala

```diff
@@ -34,15 +34,14 @@ object GasFee {
     * Calculates transaction intrinsic gas. See YP section 6.2
     *
     */
-  def calcTransactionIntrinsicGas(txData: ByteString, isContractCreation: Boolean, blockNumber: BigInt): UInt256 = {
+  def calcTransactionIntrinsicGas(txData: ByteString, isContractCreation: Boolean, blockNumber: BigInt, config: EvmConfig): UInt256 = {
```

Comment:
Should we maybe place all these methods in `EvmConfig`? Certainly the current object name `GasFee` does not sound appropriate (it never did for these methods).